src/pretix/plugins/banktransfer/payment.py

```diff
@@ -20,6 +20,7 @@ class BankTransfer(BasePaymentProvider):
     identifier = 'banktransfer'
     verbose_name = _('Bank transfer')
     abort_pending_allowed = True
+    requires_invoice_immediately = True
```

Comment:
If we have a setting for it 80 lines below, we should probably also respect that (and keep the default to ``False`` as it is, many people do not want invoices for unpaid orders)